### ISSUE TITLE
A way to power solar neutron activators at night

### ISSUE THREAD

**Issue by MattFrags** [mekanism/Mekanism-Feature-Requests#113]
**Describe the the feature you'd like**
A way to power solar neutron activators at night

**Describe alternatives you've considered**
Building 6-7 evaporation towers and try to produce so much tritium as reserves so the reactor could run on the reserve at night, but turns out anything above 2 as fuel injection rate will drain any amount of reserves in an instant.

**Additional context**
Maybe RF powered neutron activators, that would be NICE

**Comment by 238-Pu** [comment]
A better idea would be to have “neutron irradiation chambers” that can be placed inside fission reactors to absorb some of the neutron flux and make tritium from exposing lithium to that flux.

**Comment by KhanMaytok** [comment]
Just modify  two lines in the solar neutron entity file and recompile it

**Comment by zyxkad** [comment]
My idea is antimatter powered light, can emit large amount of light which can let solar things work

**Comment by Radon8472** [comment]
Maybe a cool idea would be anything that could "store" sunlight over day, and can release it at night.

Would be a cool mechanic even because it would be useable with other solar powered tools and mods

**Comment by kienerxyzzy** [comment]
> My idea is antimatter powered light, can emit large amount of light which can let solar things work

That would be too late-game, though I think I may agree with your idea. My idea is to just add an UV light that requires so much energy that you need antimatter to run it, or maybe it just takes more energy, IDK

**Comment by kienerxyzzy** [comment]
> > My idea is antimatter powered light, can emit large amount of light which can let solar things work
> 
> That would be too late-game, though I think I may agree with your idea. My idea is to just add an UV light that requires so much energy that you need antimatter to run it, or maybe it just takes more energy, IDK

Correction: that would be wrong, UV light doesn't readiate neutrons, but please allow you to get neutrons from fission and fusion(deuterium + tritium -> helium + 1 neutron)

**Comment by kienerxyzzy** [comment]
> > > My idea is antimatter powered light, can emit large amount of light which can let solar things work
> > 
> > 
> > That would be too late-game, though I think I may agree with your idea. My idea is to just add an UV light that requires so much energy that you need antimatter to run it, or maybe it just takes more energy, IDK
> 
> Correction: that would be wrong, UV light doesn't readiate neutrons, but please allow you to get neutrons from fission and fusion(deuterium + tritium -> helium + 1 neutron)

Another idea: polonium and plutonium radiates neutrons and decay
note: remember to make it return lead as output, cause lead is stable btw

**Comment by Radon8472** [comment]
> 
> Another idea: polonium and plutonium radiates neutrons and decay note: remember to make it return lead as output, cause lead is stable btw

I think this is to far away from reality.
Because plutonoium has a half-life-time of 24.110 years...
what means during ingame time it would practically never convert to lead

And further, I dont see how this could solve the request of this issues title

**Comment by zyxkad** [comment]
However, there is never reality in minecraft

**Comment by Radon8472** [comment]
> However, there is never reality in minecraft

That is true, but mekanism tries to be close to reality and making plutonium transform to lead is far away from that.

And the main point, I dont see how the request of the autor of this issue could be solved by this

**Comment by kienerxyzzy** [comment]
> > Another idea: polonium and plutonium radiates neutrons and decay note: remember to make it return lead as output, cause lead is stable btw
> 
> I think this is to far away from reality. Because plutonoium has a half-life-time of 24.110 years... what means during ingame time it would practically never convert to lead
> 
> And further, I dont see how this could solve the request of this issues title

solar neutron activator uses nutron radiation from sun, so it uses neutrons. You know what I mean.

**Comment by kienerxyzzy** [comment]
> Just modify two lines in the solar neutron entity file and recompile it

You're acting like we know how to do that
 and we actually dont

**Comment by Radon8472** [comment]
> solar neutron activator uses nutron radiation from sun, so it uses neutrons. You know what I mean.

If you focus on the "how we can get neutrons" part, I would think on the fissel reaktor....

E.g. that you add one more output, or any new device, would could collect free neutrons from the reaktor

**Comment by kienerxyzzy** [comment]
> > solar neutron activator uses nutron radiation from sun, so it uses neutrons. You know what I mean.
> 
> If you focus on the "how we can get neutrons" part, I would think on the fissel reaktor....
> 
> E.g. that you add one more output, or any new device, would could collect free neutrons from the reaktor

Another Idea: Using polonium/plutonium as a catalyst (because they don't decay that fast)

**Comment by kienerxyzzy** [comment]
> > However, there is never reality in minecraft
> 
> That is true, but mekanism tries to be close to reality and making plutonium transform to lead is far away from that.
> 
> And the main point, I dont see how the request of the autor of this issue could be solved by this

radioactive elements radiate stuff like alpha particles and neutrons. So they radiate neutrons, and you know what I mean.

**Comment by kienerxyzzy** [comment]
> > solar neutron activator uses nutron radiation from sun, so it uses neutrons. You know what I mean.
> 
> If you focus on the "how we can get neutrons" part, I would think on the fissel reaktor....
> 
> E.g. that you add one more output, or any new device, would could collect free neutrons from the reaktor

fusion reactors as well, as I said earlier in this thread:

> ...but please allow you to get neutrons from fission and fusion(deuterium + tritium -> helium + 1 neutron)
